settings.json.template

```diff
@@ -3,5 +3,10 @@
         "site" : {
             "name": "Gazelle"
         }
-    }
-}
\ No newline at end of file
+    },
+    "MAX_LOGIN_ATTEMPTS": 5,
+    "LOGIN_ATTEMPTS_EXCEEDED_TIMEOUT_ONEHOUR": 1,
+    "INVALID_LOGIN_COUNTER_TIMEOUT_ONEHOUR": 1,
+    "LOGIN_ATTEMPTS_EXCEEDED_ERRORMSG": "Maximum failed attempts reached",
```

Comment:
These error messages belong in their respective packages. Perhaps in a constants.js file?